### ISSUE TITLE
Search Results - Color contrast issues

### ISSUE THREAD

**Issue by ewlarson** [geobtaa/geoportal#519]
3. SC 1.4.3 AA – Contrast – “Leaflet” link on the bottom corner of the map has insufficient color contrast
of 4 (foreground color: #1D85B0, background color: #FCFBF8)
4. SC 1.4.3 AA – Contrast – “Click to Wake” link on the map has insufficient color contrast of 4.3
(foreground color: #76797A, background color: #FDFBFA)
5. SC 1.4.3 AA – Contrast – “Selected Page of the search results (1)” link has insufficient color contrast of
4 (foreground color: #ffffff, background color: #027BFF)

**Comment by ewlarson** [comment]
These issues have all been completed in previous a11y branch commits.
